ndscheduler/job.py

```diff
@@ -48,7 +48,7 @@ def get_failed_description(cls):
         return 'hostname: %s | pid: %s' % (hostname, pid)
 
     @classmethod
-    def get_succeeded_description(cls):
+    def get_succeeded_description(cls, result=None):
```

Comment:
Don't know if the default value is needed here as we are always passing the result to this method.